static/app/views/relocation/uploadBackup.tsx

```diff
@@ -0,0 +1,248 @@
+      return;
+    }
+
+    setFile(newFile);
+  };
+
+  const onFileUploadLinkClick = () => {
+    inputFileRef.current && inputFileRef.current.click();
+  };
+
+  const handleStartRelocation = async () => {
+    const {orgSlugs, regionUrl} = relocationOnboardingContext.data;
+    if (!orgSlugs || !regionUrl || !file) {
+      addErrorMessage(
+        t(
+          'An error has occured while trying to start relocation job. Please contact support for further assistance.'
+        )
+      );
+      return;
+    }
+    const formData = new FormData();
+    formData.set('orgs', orgSlugs);
+    formData.set('file', file);
+    formData.set('owner', user.email);
+    try {
+      await api.requestPromise(`/relocations/`, {
+        method: 'POST',
+        host: regionUrl,
+        data: formData,
+      });
+
+      addSuccessMessage('Triggered Relocation Job');
+    } catch (error) {
+      addErrorMessage(error.responseJSON?.detail);
+    }
+  };
+
+  return (
+    <Wrapper>
+      <StepHeading step={4}>
+        {t('Upload Tarball to begin the relocation process')}
+      </StepHeading>
+      <motion.div
+        transition={testableTransition()}
+        variants={{
+          initial: {y: 30, opacity: 0},
+          animate: {y: 0, opacity: 1},
+          exit: {opacity: 0},
+        }}
+      >
+        <p>
+          {t(
+            'Nearly done! The file is being uploaded to sentry for the relocation process. You can close this tab if you like. We will email  when complete.'
+          )}
+        </p>
+        {file ? (
+          <FinishedWell centered>
+            <IconFile className="file-icon" size="xl" />
+            <div>
+              <p>{file.name}</p>
+              <a onClick={() => setFile(undefined)}>{t('Remove file')}</a>
+            </div>
+            <StartRelocationButton
+              size="md"
+              priority="primary"
+              onClick={handleStartRelocation}
+              icon={<IconUpload className="upload-icon" size="xs" />}
+            >
+              {t('Start Relocation')}
+            </StartRelocationButton>
+          </FinishedWell>
+        ) : (
+          <UploadWell
+            onDragEnter={handleDragEnter}
+            onDragOver={event => event.preventDefault()}
+            onDragLeave={handleDragLeave}
+            onDrop={handleDrop}
+            centered
+            aria-label="dropzone"
+            draggedOver={dragCounter > 0}
+          >
+            <StyledUploadIcon className="upload-icon" size="xl" />
+            <UploadWrapper>
+              <p>{t('Drag and Drop file here or')}</p>
+              <a onClick={onFileUploadLinkClick}>{t('Choose file')}</a>
+              <UploadInput
+                name="file"
+                type="file"
+                aria-label="file-upload"
+                accept=".tar"
+                ref={inputFileRef}
+                onChange={e => handleFileChange(e)}
+                hidden
+                title=""
```

Comment:
nit: maybe worth filling this in for hover-text? Something like "Drag and drop a relocation tarball here"

Reply:
I played around with hover text, but I didn't think it looked very good so I scrapped it